media/js/firefox/lockwise/lockwise.js

```diff
@@ -0,0 +1,26 @@
+(function() {
+    'use strict';
+
+    var client = window.Mozilla.Client;
+
+    if (client._isFirefox() === true && client._getFirefoxMajorVersion() >= '70') {
+        document.querySelector('.for-firefox-69-and-below').remove();
+        document.querySelector('.for-non-firefox-users').remove();  
+    }
+    else if ( client._isFirefox() === true && client._getFirefoxMajorVersion() < '70' ) {
```

Comment:
Rather than repeating the call to `client._getFirefoxMajorVersion()` again here, could you store the result in a variable for reference a second time? Maybe the conditionals here could be reworked a little?